lambda/create/main.go

```diff
@@ -27,13 +33,14 @@ type Verifier interface {
 }
 
 type Lambda struct {
-	store    Store
-	verifier Verifier
-	logger   Logger
+	eventBridgeClient EventBridgeClientWrapper
```

Comment:
could name this `eventClient EventClient` since you've abstracted away from eventbridge